### PR TITLE
removed some donator items from the blueshift map

### PR DESCRIPTION
**Pull request by Useroth**

### CODE REVIEW

**Comment by Iamgoofball** [comment]
:-1:
Web edit of a map file. I can tell it's a web edit because you did it on a branch on the repository.

**Review by GoldenAlpharex — APPROVED** [review]

**Comment by Iamgoofball** [comment]
Please do not merge web edits of map files without verifying they load in a local map editor.

**Comment by GoldenAlpharex** [comment]
> 👎 Web edit of a map file. I can tell it's a web edit because you did it on a branch on the repository.

Not really, I mean, most of us maintainers make PRs from other branches of the repo since it's less of a mess to update them and such.

**Pull request merged**
